README.md

```diff
@@ -73,6 +73,41 @@ export PYTHONPATH=[path_to_repo_dir]/python:$PYTHONPATH
 
 You can also add it to environment permanently. On Ubuntu, append the line above to *~/.bashrc* and run `source ~/.bashrc`. On Mac OS X, append it to *~/.bash_profile* and run `source ~/.bash_profile`.
 
+##Usage through Docker
+
+After cloning the WMAF repo, cd to the repo directory and run:
+
+```
+docker build -t vmaf .
```

Comment:
Excuse my unfamiliarity with docker. When I run this command under mac, i got the following message:
unable to prepare context: Cannot locate Dockerfile: absDockerfile: "/Users/zli/Temp/test_vmaf/leandromoreira_vmaf/Dockerfile"